test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampProviderTests.cs

```diff
@@ -276,15 +276,15 @@ public async Task GetTimestamp_WhenTimestampSigningCertificateRevoked_ThrowsAsyn
                 timestampService,
                 (timestampProvider, request) =>
                 {
-                    var exception = Assert.Throws<TimestampException>(
-                        () => timestampProvider.GetTimestamp(request, NullLogger.Instance, CancellationToken.None));
+                    var exception = Assert.ThrowsAsync<TimestampException>(
+                        async () => await timestampProvider.GetTimestampAsync(request, NullLogger.Instance, CancellationToken.None));
 
-                    Assert.Equal("Certificate chain validation failed.", exception.Message);
+                    Assert.Equal("Certificate chain validation failed.", exception.Result.Message);
```

Comment:
With your change the type of `exception` changed from `TimestampException` to `Task<TimestampException>`; the local is no longer well-named.  `exceptionTask` would be a more suitable name.

However, there is a simpler and cleaner way to adapt to the sync -> async change you made.
```C#
    var exception = await Assert.ThrowsAsync<TimestampException>( 
        () => timestampProvider.GetTimestampAsync(request, NullLogger.Instance, CancellationToken.None)); 
```
Now `exception` is again of type `TimestampException` and you don't need to change any subsequent statements (like the change in the following `Assert.Equal(...)` statement).

Please apply this feedback throughout the file and other files where you made similar sync -> async changes.

Reply:
Thanks! Changed all the exceptionTask type I created.